library/build.gradle

```diff
@@ -11,7 +11,7 @@ apply from: '../maven_push.gradle'
 
 android {
   compileSdkVersion rootProject.ext.compileSdkVersion
-  buildToolsVersion rootProject.ext.buildToolsVersion
+  buildToolsVersion '28.0.3'
```

Comment:
I this project, I define all versions in the main top-level `build.gradle` file and it should be done this way here too. Top-level `buildToolsVersion` can be updated.